superset-frontend/cypress-base/cypress/integration/explore/control.test.ts

```diff
@@ -24,7 +24,7 @@ import { FORM_DATA_DEFAULTS, NUM_METRIC } from './visualizations/shared.helper';
 describe('Datasource control', () => {
   const newMetricName = `abc${Date.now()}`;
 
-  it('should allow edit dataset', () => {
+  it.only('should allow edit dataset', () => {
```

Comment:
Let's remove the `.only` so other tests run :)

Reply:
Oops, done!